Counter/js.js

```diff
@@ -0,0 +1,40 @@
+let num = 0;
+
+function inc1() {
+  num += 1;
+  document.getElementById("num").innerHTML = num;
+}
+
+function inc2() {
+  num += 2;
+  document.getElementById("num").innerHTML = num;
+}
+
+function inc3() {
+  num += 3;
+  document.getElementById("num").innerHTML = num;
+}
```

Comment:
```suggestion
const inc = (value) => {
  document.getElementById('num').innerHTML = num += value;
};
```
These functions are correct, but they are also very much alike. I suggest shortening the code.